config/dev.env

```diff
@@ -1,7 +1,7 @@
-CB_URL=localhost
-CB_USER=Administrator
-CB_PASS=password
-CB_BUCKET=user_profile
-IS_CAPELLA=false
+DB_CONN_STR=couchbases://<identifier>.cloud.couchbase.com
+DB_USERNAME=
+DB_PASSWORD=
+DB_BUCKET_NAME=travel-sample
+IS_CAPELLA=true
 APP_PORT=3000
-DELAY=5000
+DELAY=5000
```

Comment:
Is DELAY getting used? I would expect it to have been part of the older index & bucket creation logic.

Reply:
Removed DELAY and the corresponding delay.js